app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You don't have vaccine")
+
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Your vaccine is out of date")
+
+        if not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("YOU DON'T HAVE MASK!!!")
```

Comment:
The error message for `NotWearingMaskError` is in all caps and might not be considered a descriptive message. According to the checklist, you should use descriptive error messages. Consider rephrasing it to something like "You are not wearing a mask."